error: ‘size_t’ was not declared in this scope

Compilation error:

> In file included from /home/fi0/repositories/mantidproject/mantid0/Framework/Indexing/src/Group.cpp:7:0:
> /home/fi0/repositories/mantidproject/mantid0/Framework/Indexing/inc/MantidIndexing/Group.h:27:37: error: ‘size_t’ was not declared in this scope
>        const std::vector<std::vector<size_t>> &grouping);

Framework/Indexing/inc/MantidIndexing/Group.h
Framework/Indexing/inc/MantidIndexing/Extract.h



### Platforms affected
Ubuntu 16.04 Python 3.5

TODO:
`std::size_t`

--- a/Framework/Indexing/inc/MantidIndexing/Extract.h
+++ b/Framework/Indexing/inc/MantidIndexing/Extract.h
@@ -25,10 +25,10 @@ class SpectrumIndexSet;
 MANTID_INDEXING_DLL IndexInfo extract(const IndexInfo &source,
                                       const SpectrumIndexSet &indices);
 MANTID_INDEXING_DLL IndexInfo extract(const IndexInfo &source,
-                                      const std::vector<size_t> &indices);
+                                      const std::vector<std::size_t> &indices);
 MANTID_INDEXING_DLL IndexInfo extract(const IndexInfo &source,
-                                      const size_t minIndex,
-                                      const size_t maxIndex);
+                                      const std::size_t minIndex,
+                                      const std::size_t maxIndex);
 
 } // namespace Indexing
 } // namespace Mantid

--- a/Framework/Indexing/inc/MantidIndexing/Group.h
+++ b/Framework/Indexing/inc/MantidIndexing/Group.h
@@ -24,7 +24,7 @@ class IndexInfo;
 */
 MANTID_INDEXING_DLL IndexInfo
 group(const IndexInfo &source, std::vector<SpectrumNumber> &&specNums,
-      const std::vector<std::vector<size_t>> &grouping);
+      const std::vector<std::vector<std::size_t>> &grouping);
 
 } // namespace Indexing
 } // namespace Mantid

